.eslintrc.json

```
@@ -0,0 +1,31 @@
+{
+    "env": {
+        "browser": true,
```

```suggestion
        "node": true,
```

Could also add `"jest": true,` if you're going to do unit testing.